src/wizards/topics.ts

```diff
@@ -7,13 +7,17 @@ import { MultiStepInput, showErrorMessage, State } from "./multiStepInput";
 import { validateTopicName, validatePartitions, validateReplicationFactor } from "./validators";
 
 const DEFAULT_PARTITIONS = 1;
-const DEFAULT_REPLICATION_FACTOR = 1;
+
+const REPLICATION_FACTOR_KEY = 'offsets.topic.replication.factor';

```

Comment:
Those constants are about broker configs (https://kafka.apache.org/documentation/#brokerconfigs). I will create a `src/client/configs.ts` file and define those constants with namespace (liek https://github.com/redhat-developer/vscode-xml/blob/27983849c3d19b7a53e29e66a4f128c1e09275d5/src/commands/commandConstants.ts#L16) :

```ts
/**
 * @see https://kafka.apache.org/documentation/#brokerconfigs
 */
export namespace BrokerConfigs {

    /**
    * @see https://kafka.apache.org/documentation/#brokerconfigs_default.replication.factor
    */
    export const DEFAULT_REPLICATION_FACTOR = 'default.replication.factor';

    /**
     * @see https://kafka.apache.org/documentation/#brokerconfigs_offsets.topic.replication.factor
     */
    export const OFFSETS_TOPIC_REPLICATION_FACTOR = 'offsets.topic.replication.factor';

}

```

Reply:
done